src/main/kotlin/app/batch/S3Reader.kt

```diff
@@ -97,16 +97,29 @@ class S3Reader(private val s3client: AmazonS3, private val keyPairGenerator: Key
         logger.info("Found objects in s3", "s3_objects_found", "s3://$bucketName/$fullPrefix", "s3_location", "${objectSummaries.size}")
         val objectSummaryKeyMap = objectSummaries.map { it.key to it }.toMap()
         val keyPairs =
-            keyPairGenerator.generateKeyPairs(objectSummaries.map { it.key },
-                fileNameFormat.toRegex(),
-                fileNameFormatDataExtension.toRegex(),
-                fileNameFormatMetadataExtension.toRegex())
-
-        val pairs = keyPairs.map {
-            val obj = objectSummaryKeyMap[it.dataKey]
-            val meta = objectSummaryKeyMap[it.metadataKey]
-            S3ObjectSummaryPair(obj, meta)
-        }
+                keyPairGenerator.generateKeyPairs(objectSummaries.map { it.key },
```

Comment:
why has the indentation changed?

Reply:
Looks right to me now, inline with the others

Reply:
how so? The rest of the file is indented less

Reply:
is it because you should indent broken lines?

Reply:
this was almost certainly intellij reformatting on commit.